users/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+from .models import UserProfile
+from django.utils.translation import ugettext_lazy
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = ['username', 'email']
+
+
+class CreateUserSerializer(serializers.ModelSerializer):
```

Comment:
To force user to use secure password, we can user Django's password_validation method (`import django.contrib.auth.password_validation`).

We can use password_validation in `field level validation`. https://www.django-rest-framework.org/api-guide/serializers/#field-level-validation

Reply:
In other way, you can use `validators` parameter of field.